lib/modules/datasource/dotnet-sdk/index.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { getPkgReleases } from '..';
+import { Fixtures } from '../../../../test/fixtures';
+import * as httpMock from '../../../../test/http-mock';
+import { EXTERNAL_HOST_ERROR } from '../../../constants/error-messages';
+import { DotnetSdkDatasource } from '.';
+
+const releasesIndex = Fixtures.getJson('releases-index.json');
+const releases7_0 = Fixtures.getJson('releases-7.0.json');
+const releases6_0 = Fixtures.getJson('releases-6.0.json');
+const releases5_0 = Fixtures.getJson('releases-5.0.json');
+const releases3_1 = Fixtures.getJson('releases-3.1.json');
```

Comment:
can we inline those and reduce to minimum?

because IO is expensive on CI (especially on windows)

Reply:
I can definitely reduce them down to the properties we need, but even still they are relatively large. I want to keep them as separate files.

Reply:
I am stripping out all unnecessary properties using `jq`. For posterity, here's the (monstrous 😱) `jq` filter I used:

```
. | 
{
    "channel-version",
    "latest-release",
    "latest-release-date",
    "latest-runtime",
    "latest-sdk",
    "support-phase",
    releases: [.releases[] |
    { 
        "release-date",
        "release-version",
        security,
        "release-notes",
        runtime: {
            "version": .runtime.version,
            "version-display": .runtime."version-display"
        },
        sdk: {
            "version": .sdk.version,
            "version-display": .sdk."version-display"
        },
        
    }]
}
```